src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/Overview.scss

```diff
@@ -0,0 +1,171 @@
+@import 'src/styles/common';
+
+.titleWithSeparator {
```

Comment:
`withSeparator` is redundant (there are no other titles in this component).
Also, maybe call it sectionHead?

Reply:
That's done